test/vault_helpers.go

```diff
@@ -102,11 +102,9 @@ func deployCluster(t *testing.T, amiId string, awsRegion string, examplesDir str
 	terraformOptions := &terraform.Options{
 		TerraformDir: examplesDir,
 		Vars: mergeMaps(terraformVars, map[string]interface{}{
-			VAR_AMI_ID:                 amiId,
-			VAR_VAULT_CLUSTER_NAME:     fmt.Sprintf("vault-test-%s", uniqueId),
-			VAR_CONSUL_CLUSTER_NAME:    fmt.Sprintf("consul-test-%s", uniqueId),
```

Comment:
had to move this out since the default behaviour is to use consul, being able to use this for dynamo means these variables need to move  closer to the test cases using them